lib/modules/datasource/dotnet/__fixtures__/releases-3.1.json

```diff
@@ -0,0 +1,472 @@
+{
+  "channel-version": "3.1",
+  "latest-release": "3.1.29",
+  "latest-release-date": "2022-09-13",
+  "latest-runtime": "3.1.29",
+  "latest-sdk": "3.1.423",
+  "support-phase": "maintenance",
+  "releases": [
```

Comment:
i think we can reduce these files some more and only leave a couple of versions (eg 3 or 5)

Reply:
I cut each fixture down and tried to include:
- First and last `preview` release
- One `rc` release
- First and last GA release